documentation/writing_plans.md

```diff
@@ -224,6 +224,80 @@ plan mymodule::myplan {
 }
 ```
 
+## Debugging plans
+
+By default, Bolt does not print verbose logs for each plan execution to stdout.
+However, you can use one of the following methods to investigate a plan
+execution:
+- Each time you run a Bolt command, Bolt prints a debug log to a
+  `bolt-debug.log` file in the root of your project directory.
+- You can use the `--verbose` CLI option for verbose logging to stdout.
+- You can print a parameter to stdout using the `out::message` function. 
+- You can adjust your log level for detailed information on how Bolt is
+  executing your plan.
+
+### Using `out::message` to debug a plan
+
+You can print any plan parameter to stdout using the `out::message` function. If
```

Comment:
```suggestion
You can print anything to stdout using the `out::message` function. If
```

Reply:
I mostly think this is useful because I frequently just print messages like "Got here" or other non-variable data to help debug plans.

Reply:
Alternatively you might be looking for the word 'variable'?

Reply:
Ah you're right - I think I meant variable.